ansible/roles/kstest/files/authorized_keys/kstests.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDJjCafb/FZ4hIuQXfUPvhkZmQT2na//fl2rmnG5i65EyJXrWZyO2VM4n4rDmYT2scxp3BDV1ovacTetiqUW3cBID+pzuyVmRonsE6eNudlP47BWv14kPY3SocImt4lgjA2bag4yhgN7mx20c3bLGL7L25D27SZwDYfBbPUl1uuypmMlyiLZ7Cwpf+qEeobfk5nwjZMDZEL9abPk2HYIm+2udLCqUtVj/51lQ+XL5AH1EJvMdh4Fv6GuqV69jSZoc9XcPoUX22NXIirym0Tpzro/j4xOZ0jDN5E012fdMcKwFwLMiXrSq1Amgk4B7z0EQQ4z3/by/EAsxCEkyuoa5vr Generated-by-Nov
```

Comment:
Again, we don't want to have SSH key here, even when it is public.
Ask this question: What is the benefit for someone who checked out this repo and is outside of our team.